### DISCUSSION TITLE
Configure Table with REST API Data Source

### DISCUSSION THREAD

**Discussion by kodecrash** [Budibase/budibase#6379] [Help] [answered]
I have configured REST API data source which fetches users from below API.

**https://jsonplaceholder.typicode.com/users**

![image](https://user-images.githubusercontent.com/8296994/174445147-16f84ee8-5ff5-44c4-bcc3-307fc61cda61.png)

I am using this data source to populate users in a table component. But after previewing application it is fetching data but unable to display it in the table. How to configure table columns in this case.

Unable to see columns in below settings of table component.

![image](https://user-images.githubusercontent.com/8296994/174445166-0d409902-d393-4890-a919-14ffe004ab08.png)

**Comment by melohagan** [comment, marked as answer]
Hi @kodecrash 

Could you show some screenshots of your component tree please? Maybe you just need to try removing and re-adding the table/data provider.
You don't have any other code in the transformer either?

Thanks for providing the API URL - I was able to get it to display in a table fine:
![Screenshot 2022-06-20 at 10 17 29](https://user-images.githubusercontent.com/101575380/174569099-e3d82f8f-f78d-405f-89b7-0e8f5bd0ab78.png)
![Screenshot 2022-06-20 at 10 17 49](https://user-images.githubusercontent.com/101575380/174569162-fe519457-cbbb-4f78-8420-6dab56283d9d.png)
![Screenshot 2022-06-20 at 10 18 06](https://user-images.githubusercontent.com/101575380/174569204-b2822892-716a-436b-a32d-73649131155c.png)

**Reply by kodecrash** [reply]
It started working now. Thanks.
